hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,6 +1,24 @@
 # How to boot from a USB Mass Storage Device on a Raspberry Pi 3
 This tutorial explains how to boot your Raspberry Pi 3 from a USB mass storage device such as a flash drive or USB hard disk. Be warned that this feature is experimental and may not work with all USB mass storage devices.
 
+## Before Starting
+Check to make sure your SD card partitions are right for this example:
```

Comment:
I see that now, I'll get it taken care of here soon!  (see comment below)  I'm new to github, but I'll catch on.